src/merkle/merkle-objects.ts

```diff
@@ -10,8 +11,13 @@ import { IpfsService } from '../services/ipfs-service.js'
 import { BloomFilter } from 'bloom-filters'
 import { StreamID } from '@ceramicnetwork/streamid'
 
+const packageJson = JSON.parse(
+  fs.readFileSync(new URL('../../node_modules/bloom-filters/package.json', import.meta.url), 'utf8')
```

Comment:
this seems reasonable to me but I'd like @ukstv to take a look and make sure this is robust enough and will work in all the different environments in which anyone might want to run a CAS.  For example will this work on Windows?  Is anything different required after a move to ES2020/ESM?

Reply:
@stbrody Confirm. This thing will work on Windows.

Reply:
I mean, I did run similar code on Windows, and it worked.